package.json

```diff
@@ -6,7 +6,7 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "cypress run",
-    "test:e2e": "start-server-and-test serve http://localhost:8080 cypress:run",
+    "test:headless": "start-server-and-test serve http://localhost:8080 cypress:run",
```

Comment:
Is it possible to read `:8080` port from `ENV`? like this it could easily be overwritten if `8080` is not available...

Reply:
should be possible with `PORT` variable, for instance `PORT=9999 npm run test:headless`

Reply:
but I wasn't able to do a full auto port pick. If `PORT` or default port are being used, tests will try to use what was given while `npm run serve` will choose a new port. This will fail tests most likely...

Reply:
Hmm, serving is done at the port specified with `PORT=9999`, but the testing still seems to wait for readyness at `8080`:
```

> web-mapviewer@0.1.0 test:headless /home/ltboc/src/web-mapviewer
> start-server-and-test serve http://localhost:8080 cypress:run

1: starting server using command "npm run serve"
and when url "http://localhost:8080" is responding with HTTP status code 200
running tests using command "npm run cypress:run"


> web-mapviewer@0.1.0 serve /home/ltboc/src/web-mapviewer
> vue-cli-service serve

 INFO  Starting development server...
98% after emitting CopyPlugin

 DONE  Compiled successfully in 3426ms                                                                                                                                                              4:18:07 PM


  App running at:
  - Local:   http://localhost:9999/ 
  - Network: http://192.168.1.128:9999/

  Note that the development build is not optimized.
  To create a production build, run npm run build.

^C
```

Reply:
I runs test on port 9999 for me if I do `PORT=9999 npm run test:headless` but I haven't tried to set the env variable for the session globally, is that what you did?

Reply:
Nope, just like you... for the app it is using port `9999` (see "App running at", but at the top, it's saying "... and when url 'http://localhost:8080'" it's still mentioning `8080`)